qemu/tests/block_hotplug.py

```diff
@@ -109,11 +109,10 @@ def unplug_device(vm, get_disk_cmd, device):
                     controller = qdevices.QDevice(controller_model, params={"id":
                                                   "hotadded_scsi%s" % num})
                     bus_extra_param = params.get("bus_extra_params_%s" % img_list[num + 1])
-                    # TODO:Add iothread support for qdevice
-                    if bus_extra_param and "iothread" in bus_extra_param:
-                        match = re.search("iothread=(\w+)", bus_extra_param)
-                        if match:
-                            qdevice_params = {"iothread": match.group(1)}
+                    if bus_extra_param:
```

Comment:
I did not see `bus_extra_param` in the cfg, just want check with you, what kind of extra_param it can be? The previous code only adds `iothread`  or does nothing.
But here it will add what all extra_param as long as it in 'xxx=xxx' patten.

Reply:
hachen, bus_extra_param is defined in the last line of block_hotplug.cfg, at present it just means 'bus_extra_params_stg0 = "iothread=${iothreads}"'.  If we have new (key, value) for bus_extra_param latter, we don't need to modify the code again.